src/callbacks_step/amr_dg2d.jl

```diff
@@ -20,7 +20,7 @@ function rebalance_solver!(u_ode::AbstractVector, mesh::TreeMesh{2}, equations,
   # Retain current solution data
   old_n_elements = nelements(dg, cache)
   old_cell_ids = copy(cache.elements.cell_ids)
-  old_u_ode = copy(u_ode)
+  old_u_ode = local_copy(u_ode)
```

Comment:
Why can't we use a regular copy here?

Reply:
OK!

Reply:
Yes, we can get rid of `local_copy` :slightly_smiling_face: looks like some other changes I made after introducing this workaround fixed the underlying problem.